spec.html

```diff
@@ -37630,8 +37638,7 @@ <h1 id="ao-issuperreference">IsSuperReference ( _V_ )</h1>
       <p>The abstract operation ValueOfReadEvent takes two arguments, a candidate execution _execution_ and a ReadSharedMemory or ReadModifyWriteSharedMemory event _R_. It performs the following steps:</p>
       <emu-alg>
         1. Assert: _R_ is a ReadSharedMemory or ReadModifyWriteSharedMemory event.
-        1. Let reads-bytes-from be _execution_.[[ReadsBytesFrom]].
-        1. Let _Ws_ be reads-bytes-from(_R_).
+        1. Let _Ws_ be _execution_.[[ReadsBytesFrom]](_R_).
```

Comment:
There used to be 20 occurrences of the former notation, but then 3b50623 replaced all but one of them (this one) with the latter notation. I suspect it was simply overlooked.

Note that 'Coherent Reads' has a step that's identical to the latter notation.